packages/sanity/src/core/i18n/bundles/studio.ts

```diff
@@ -653,6 +653,13 @@ export const studioLocaleStrings = defineLocalesResources('studio', {
     'Adjust the rectangle to crop image. Adjust the circle to specify the area that should always be visible.',
   /** Error: `{{errorMessage}}` */
   'inputs.imagetool.load-error': 'Error: {{errorMessage}}',
+  /** SVG images are not adjusted for hotspot and crop when served from the Sanity image. If you want the below hotspot and crop settings to apply, make sure to append `fp=jpg` to the image url, or call `format('jpg')` if using `@sanity/image-url` */
+  'inputs.imagetool.svg-warning.developer-info': `SVG images are not adjusted for hotspot and crop when served from the Sanity image API. If you want the below hotspot and crop settings to apply, make sure to append  <code>fp=jpg</code> to the <ImageUrlDocumentationLink>image url</ImageUrlDocumentationLink>, or call <code>format('jpg')</code> if using <ImageUrlPackageDocumentationLink>@sanity/image-url</ImageUrlPackageDocumentationLink>`,
```

Comment:
Tiny thing: We should probably add a period to the end of this sentence for consistency.

Reply:
Thanks for pointing out my poor writing and silly mistakes! (I'm hopeless at typing when I don't have a linter yelling at me 😄)

I'll address these, but think @kmelve had some suggestions about this too, so awaiting his feedback before fixing.